lms/envs/test.py

```diff
@@ -54,62 +41,26 @@
 for log_name, log_level in LOG_OVERRIDES:
     logging.getLogger(log_name).setLevel(log_level)
 
-# mongo connection settings
-MONGO_PORT_NUM = int(os.environ.get('EDXAPP_TEST_MONGO_PORT', '27017'))
-MONGO_HOST = os.environ.get('EDXAPP_TEST_MONGO_HOST', 'localhost')
-
-THIS_UUID = uuid4().hex[:5]
-
-DISABLE_SET_JWT_COOKIES_FOR_TESTS = True
-
 # can't test start dates with this True, but on the other hand,
 # can test everything else :)
 DISABLE_START_DATES = True
 
-# Most tests don't use the discussion service, so we turn it off to speed them up.
-# Tests that do can enable this flag, but must use the UrlResetMixin class to force urls.py
-# to reload. For consistency in user-experience, keep the value of this setting in sync with
-# the one in cms/envs/test.py
-ENABLE_DISCUSSION_SERVICE = False
-
-ENABLE_SERVICE_STATUS = True
-
 ENABLE_VERIFIED_CERTIFICATES = True
 
-# Toggles embargo on for testing
-EMBARGO = True
-
-# Enable the milestones app in tests to be consistent with it being enabled in production
-MILESTONES_APP = True
-
-ENABLE_ENROLLMENT_TRACK_USER_PARTITION = True
-
 ENABLE_BULK_ENROLLMENT_VIEW = True
 
 ENABLE_BULK_USER_RETIREMENT = True
 
-DEFAULT_MOBILE_AVAILABLE = True
-
-# Need wiki for courseware views to work. TODO (vshnayder): shouldn't need it.
-WIKI_ENABLED = True
-
-# Enable a parental consent age limit for testing
-PARENTAL_CONSENT_AGE_LIMIT = 13
+COMMON_TEST_DATA_ROOT = COMMON_ROOT / "test" / "data"
 
-# Local Directories
-TEST_ROOT = path("test_root")
-# Want static files in the same dir for running on jenkins.
-STATIC_ROOT = TEST_ROOT / "staticfiles"
 WEBPACK_LOADER['DEFAULT']['STATS_FILE'] = STATIC_ROOT / "webpack-stats.json"
 WEBPACK_LOADER['DEFAULT']['LOADER_CLASS'] = 'webpack_loader.loader.FakeWebpackLoader'
```

Comment:
Flagging the same webpack settings as I mentioned in the `cms/envs/test.py`, this can be moved up as a derived setting and removed.